setup.py

```diff
@@ -32,6 +32,9 @@
 invenio_search_version = '>=1.2.0,<1.3.0'
 
 extras_require = {
+    'badges': [
+        'CairoSVG>=1.0.20',
+    ],
```

Comment:
curious: what makes the cut for an `extras_require` and what doesn't?

Reply:
AFAIK usually thinks that might be optional or choice (e.g. ES versions). But it's true in this case it fails to build otherwise. I was just thinking that not all UIs might want the badges... but it's just an assumption lets see what happens in reality

Moved to main install requirements